android/app/src/main/kotlin/net/defined/mobile_nebula/Sites.kt

```diff
@@ -251,6 +251,7 @@ class IncomingSite(
     val cipher: String,
     val sortKey: Int?,
     var logVerbosity: String?,
+    val logWrap: Boolean?,
```

Comment:
We can undo this and it's friends now that it isn't a side based config

Reply:
Fixed! ✅